model/components/kieker.yml

```diff
@@ -0,0 +1,90 @@
+---
+components:
+
+# kieker monitoring
+  - id: kieker-monitoring
+    name: Kieker Monitoring
+    description: Kieker Monitoring component for software instrumentation, performance data collection, and logging.
+    link: http://kieker-monitoring.net/
+    github:
+      - kieker-monitoring/kieker
+    twitter: kiekerapm 
+    logo: kieker
+    license:
+      - Apache License 2.0
+    categories:
+      - agent
+      - libraries
+      - collector
+      - storage
+      - pipeline
+    capabilities:
+      tech:
+        - java
+      aspects:
+        - tracing
+        - metrics
+    connections:
+      - influx-db
+      - kafka
+      - kieker-analysis
+      - kieker-trace-diagnosis-gui
+      - kieker-trace-analysis-ui
+
+# kieker analysis
+  - id: kieker-analysis
+    name: Kieker Analysis
+    description: Kieker Analysis component for reading, analyzing, and visualizing the monitoring data. It manages the lifecycle of various analysis plugins.
+    link: http://kieker-monitoring.net/
+    github:
+      - kieker-monitoring/kieker
```

Comment:
Do you have additional github repositories for the analysis plugins? if, yes you can add all of them here. The github Property is a list, so you can have multiple per component.

Reply:
No. It's all a part of this one repository.

Only trace diagnosis GUI is in another repo.